### DISCUSSION TITLE
I have question

### DISCUSSION THREAD

**Discussion by XiaJack** [fallahn/tmxlite#171] [Q&A]
![Trouble](https://github.com/user-attachments/assets/15ee48de-13f1-4a4d-80ef-fb530f3a6231)
How does it work? It's been going on for days

**Comment by fallahn** [comment]
If you're building the demo you need to build the tmxlite library first. Then you need to add the directory it's in to the search paths for the CMake file for the demo builder. You can use CMake's link_directories() for this:
https://cmake.org/cmake/help/latest/command/link_directories.html

**Comment by XiaJack** [comment]
> 如果您正在构建演示，则需要先构建 tmxlite 库。然后，您需要将其所在的目录添加到演示构建器的 CMake 文件的搜索路径中。您可以使用 CMake 的 link_directories() 来实现此目的： [https: //cmake.org/cmake/help/latest/command/link_directories.html](https://cmake.org/cmake/help/latest/command/link_directories.html)

I've compiled tmxlite, and Write the correct path,But there is a problem
![240825](https://github.com/user-attachments/assets/fe452d11-0dc9-42f9-9f95-ea213946aa5d)
